tfrecorder/beam_pipeline.py

```diff
@@ -253,8 +227,7 @@ def build_pipeline(
   Note: These inputs must be validated upstream (by client.create_tfrecord())
   """
 
-  job_name = _get_job_name(job_label)
-  job_dir = _get_job_dir(output_dir, job_name)
+  _, job_name = os.path.split(job_dir)
```

Comment:
Isn't this going to break beam?  I think each job has to have a unique name, right?

Reply:
`job_dir` is actually something like `/path/to/output_dir/job_name` and `job_name` will be parsed out. `job_name` is the unique tfrecord directory generated by the pipeline.